### PR TITLE
Feature/news real data

### PR DESCRIPTION
**Pull request by arealclimber**
### DEVELOP

- [x] NewsArticle
- [x] MarketSection
- [x] TradePageBody
- [x] /pages/news/[newsId].tsx
- [x] /pages/trade/cfd/[tickerId].tsx

### CHECK LIST
 
- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 0
- new Class / Component:  0
- new loop: 0
- new recursive: 0
- high risk: 0
- new sql: 0

### UML

- [Sequence Diagram 001 (interaction between user, frontend and backend)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00001.md)
- [Sequence Diagram 002 (background mechanism)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00002.md)
- [Component Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCP00001.md)
- [Class Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCD00001.md)
- [Global context document](https://www.notion.so/Global-Context-public-d93ac7c941c94fb6b173d34a7bc4e959)
- [Table of interface](https://github.com/CAFECA-IO/EventReport/blob/main/meeting/20230106_tbd_mvp.md)

### CODE REVIEW

File: src/components/trading_input/trading_input.tsx [hunk 1]
```diff
@@ -60,7 +60,8 @@ const TradingInput = ({
     typeof setTimeout
   > | null>(null);
 
-  const regex = /^(0(\.0)?|[1-9]\d*(\.\d{0,2})?|0\.\d*[1-9]\d*)$/;
+  // const regex = /^(0(\.0)?|[1-9]\d*(\.\d{0,2})?|0\.\d*[1-9]\d*)$/;
```
Comment (arealclimber) [inline src/components/trading_input/trading_input.tsx:63]: comment 刪掉

File: src/components/crypto_news_section/crypto_news_section.tsx [hunk 2]
```diff
@@ -3,31 +3,40 @@ import Link from 'next/link';
...
+
+const CryptoNewsSection = (props: ICryptoNewsSectionProps) => {
   const {t}: {t: TranslateFunction} = useTranslation('common');
 
   const overallWidth = 'w-full pr-5 lg:p-0 lg:w-2/3 xl:w-3/4';
   const dividerWidth = 'w-full lg:w-2/3 xl:w-3/4';
   const marketCtx = useContext(MarketContext);
 
-  const cryptoBriefNews = marketCtx.tickerStatic?.cryptoBriefNews ?? [];
+  // const cryptoBriefNews = marketCtx.tickerStatic?.cryptoBriefNews ?? [];
```
Comment (arealclimber) [inline src/components/crypto_news_section/crypto_news_section.tsx:21]: comments 刪掉

**Review by arealclimber — COMMENTED** [review]
comments

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**